core/server/services/posts/posts-service.js

```diff
@@ -17,69 +17,35 @@ class PostsService {
     }
 
     async editPost(frame) {
-        let model;
-
-        // Make sure the newsletter_id is matching an active newsletter
-        if (frame.options.newsletter_id) {
-            const newsletter = await this.models.Newsletter.findOne({id: frame.options.newsletter_id, filter: 'status:active'}, {transacting: frame.options.transacting});
```

Comment:
Did this ever work? 😬

Reply:
No, we didn't have tests for this 😬 We even tested with archived newsletters without knowing 😅